lib/shared/config/env-flags.js

```diff
@@ -34,11 +34,10 @@ function mergeConfigToEnvFlags(env, config, cliOpts) {
       return [].concat(envInfo.value, configInfo.value);
     }
 
+    /* istanbul ignore else */
     if (envInfo.keyChain === 'nodeFlags') {
-      return [].concat(configInfo.value || []);
+      return [].concat(configInfo.value);
```

Comment:
If `configInfo.value` is `undefined`, an `undefined` value will be concat'ed into the array.

Reply:
We should add a test for that.

Reply:
Looks like a test was added.